src/components/CameraPermissions/Primer/style.css

```diff
@@ -21,20 +23,19 @@
 
 .image {
   font-size: @font-size-small;
-
-  border-radius: 3*@unit-small;
-  border: 1px solid @color-background;
-  margin: 0 auto 8*@unit-small;
+  margin-top: 40*@unit;
   display: flex;
   flex-direction: column;
-  max-width: 280*@unit-small;
+  max-width: 105.01*@unit-x-small;
```

Comment:
How was this value calculated?

Reply:
105.01 is the value on abstract. The unit small was not actually getting the right value on screen though but unit-x-small was